### PR TITLE
fix relevance-based ranking for java proposals to be based on rank score

### PR DESCRIPTION
**Pull request by martinlippert**

### CODE REVIEW

**Review by BoykoAlex — APPROVED** [review]

**Comment by mickaelistria** [comment]
I have not been involved in the development of the org.eclipse.lsp4e.jdt bundle, nor is it part of a stack that I am employed to maintain (as far as I know...), so I won't be reviewing here because my review wouldn't be so useful. Both your expertises it significant enough for me trust you fully if you want to merge.

**Comment by martinlippert** [comment]
Marged, based in the positive review from @BoykoAlex and the comment from @mickaelistria . Thanks again to both of you!

File: org.eclipse.lsp4e.jdt/src/org/eclipse/lsp4e/jdt/LSJavaCompletionProposalComputer.java [hunk 1]
```diff
@@ -84,19 +84,18 @@ private ICompletionProposal[] asJavaProposals(CompletableFuture<ICompletionPropo
 		// We assume that the original proposals are in the correct order, so we set relevance
 		// based on this existing order. Note that based on IJavaCompletionProposal javadoc,
 		// relevance values are [0,1000] so we start at 1000
```
Comment (sebthom) [inline org.eclipse.lsp4e.jdt/src/org/eclipse/lsp4e/jdt/LSJavaCompletionProposalComputer.java:86]: Shouldn't this comment be removed too?

Comment (martinlippert) [reply org.eclipse.lsp4e.jdt/src/org/eclipse/lsp4e/jdt/LSJavaCompletionProposalComputer.java:86]: Yes, absolutely. Will do that asap.

**Pull request merged**